css/Components/ShowMore.scss

```diff
@@ -37,7 +42,12 @@
   max-height: 400px;
   pointer-events: none;
 
-  background-image: linear-gradient(to bottom, rgba(8, 19, 64, 0), #081340);
+  background-image: linear-gradient(to bottom, rgba(8, 19, 64, 0), $Color-deep-cove);
+
+  @include variablesSupported {
+    background-image: linear-gradient(to bottom, rgba(8, 19, 64, 0), var(--color-background));
```

Comment:
the variable name needs to change here as well

Reply:
As spoken in person, it shouldn't